server/hotrod/src/main/java/org/infinispan/server/hotrod/Decoder2x.java

```diff
@@ -241,6 +241,7 @@ private Response createResponse(HotRodHeader h, OperationStatus st, byte[] prev)
             case Success:
                switch (h.op) {
                   case PUT:
+                  case PUT_MULTIMAP:
```

Comment:
I don't think it belongs here.

Reply:
this is  a merge error

Reply:
not even a merge error ! I put that there a lot time ago, nice catch